corehq/apps/domain/forms.py

```diff
@@ -1237,6 +1237,9 @@ def current_contact_emails(self):
             ).emails
         except BillingContactInfo.DoesNotExist:
             return None
+        except AttributeError:
+            # current_subscription is None. cf. FB 168555
+            return None
```

Comment:
I think this is a little heavy-handed, and could possibly mask other bugs since an attribute error could be raised by many things happening in the try block. In this case the problem was that `self.current_subscription` was `None`. I think it would make more sense to do this at the beginning of the function:
```py
if self.current_subscription is None:
    return None
```